confdb/aclocal_cc.m4

```diff
@@ -403,6 +403,56 @@ else
 fi
 ])
 
+dnl
+dnl PAC_CC_VENDOR:
+dnl Try to get a version string for the C compiler.  We may
+dnl need this to find likely command-line arguments for accessing
+dnl shared libraries
+dnl
+AC_DEFUN([PAC_CC_VENDOR],[
+AC_MSG_CHECKING([for C compiler vendor])
+# This is complicated by some compilers (such as the Intel 8.1 ifort)
+# that return a non-zero status even when they accept the -V option
+# (a zero status is returned only if there is a file).
+pac_cv_cc_vendor="unknown"
+for arg in --version -V -v ; do
+    rm -f conftest.txt
+    PAC_RUNLOG([$CC $arg </dev/null >conftest.txt 2>&1])
+    # Ignore the return code, because some compilers set the
+    # return code to zero on invalid arguments and some to
+    # non-zero on success (with no files to compile)
+    if test -f conftest.txt ; then
+        if grep 'Portland Group' conftest.txt >/dev/null 2>&1 ; then
+            pac_cv_cc_vendor=pgi
+        elif grep 'Sun Workshop' conftest.txt >/dev/null 2>&1 ; then
+            pac_cv_cc_vendor=sun
+	elif grep 'Sun C' conftest.txt >/dev/null 2>&1 ; then
```

Comment:
What happened to the indentation? Does your editor fail you? 😄

Reply:
Copy/pasting from the Fortran version with a mix of tabs and spaces failed me...